dbms/src/AggregateFunctions/AggregateFunctionSum.h

```diff
@@ -278,10 +278,10 @@ struct AggregateFunctionSumKahanData
 };
 
 
-#define DEFAULT_DECIMAL_INFER                                                          \
-    static std::pair<PrecType, ScaleType> decimalInfer(PrecType prec, ScaleType scale) \
-    {                                                                                  \
-        return SumDecimalInferer::infer(prec, scale);                                  \
+#define DEFAULT_DECIMAL_INFER                                                           \
+    static std::tuple<PrecType, ScaleType> decimalInfer(PrecType prec, ScaleType scale) \
```

Comment:
should we also change the `std::pair` here?
https://github.com/pingcap/tiflash/blob/b27d857078b1d774da24e29630eca67316ec2b35/dbms/src/AggregateFunctions/AggregateFunctionSum.h#L294-L301

Reply:
done